packages/babel-plugin-flow-to-typescript/src/visitors/Program.ts

```diff
@@ -12,9 +12,15 @@ export const Program = {
       firstNode.leadingComments &&
       firstNode.leadingComments.length
     ) {
-      const commentIndex = firstNode.leadingComments.findIndex(
-        item => item.value.trim() === '@flow'
-      );
+      const commentIndex = firstNode.leadingComments.findIndex(item => {
+        const trimmedValue = item.value.trim();
+
+        return (
+          trimmedValue === '@flow' ||
+          trimmedValue === '@flow strict' ||
+          trimmedValue === '@flow strict-local'
+        );
+      });
```

Comment:
can you add more tests to cover this? - I guess this might not work when using recast

also, while on it - can you also add support for multiline comments, like this:

```
/**
* Some boring copyright info
* @flow
*/
```